working_time/working_time/report/expected_and_actual_working_time/expected_and_actual_working_time.js

```diff
@@ -30,4 +32,32 @@ frappe.query_reports["Expected and Actual Working Time"] = {
 			reqd: 1,
 		},
 	],
+	// fetch filter values for employee and daily_working_hours using db call and python script
+	onload: function(report) {
+		frappe.call({
+			method: "working_time.working_time.report.expected_and_actual_working_time.get_filter_values.get_employee_working_hours",
+			args: {
+				"user": frappe.session.user
+			},
+			callback: (r) => {
+				if (r.message) {
+					var filter = report.get_filter("daily_working_hours");
+					filter.set_value(r.message);
+				}
+			}
+		});
+
+		frappe.call({
+			method: "working_time.working_time.report.expected_and_actual_working_time.get_filter_values.get_employee_name",
+			args: {
+				"user": frappe.session.user
+			},
+			callback: function(r) {
+				if (r.message) {
+					var filter = report.get_filter("employee");
+					filter.set_value(r.message);
+				}
+			}
+		});
```

Comment:
```suggestion
		frappe.call({
			method: "working_time.working_time.report.expected_and_actual_working_time.get_filter_values.get_employee_name",
			callback: function(r) {
				if (r.message) {
					var filter = report.get_filter("employee");
					filter.set_value(r.message);
				}
			}
		});
```

- Doesn't need to pass the current user, it's known in the backend.